go.mod

```
@@ -1,9 +1,10 @@
-module github.com/nadeemjamali/sqs-prometheus-exporter
+module github.com/candisio/sqs-prometheus-exporter
 
 go 1.13
 
 require (
 	github.com/aws/aws-sdk-go v1.30.6
 	github.com/go-co-op/gocron v0.1.0
+	github.com/nadeemjamali/sqs-prometheus-exporter v0.1.0 // indirect
```

@candeemis I believe this would also change due to the repository move but then again I am not sure why the code would require itself. 🤔 

```suggestion
	github.com/candisio/sqs-prometheus-exporter v0.1.0 // indirect
```

---

@nagelflorian It was added automatically by Go builder/compiler, therefore I didn't remove it. I just changed it to the new value.

---

@candeemis have you tried running `go mod tidy` to prune any no-longer-needed dependencies from these files?

---

@nagelflorian it worked 👍 thanks for the advice.